core/ecschema-metadata/src/@types/ulp/index.d.ts

```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+declare module "ulp" {
```

Comment:
Looks like publish the same/similar definition [here](https://github.com/jcoreio/ulp/blob/master/lib/index.js.flow).  Is there not a types package anywhere?

We should at the very least leave a GH Issue on their repo asking if not.

Reply:
This ulp package is used to compare floats. Will experiment with almost-equal [https://www.npmjs.com/package/almost-equal](url) to see if it can accomplish the same thing. That repo is typed and a bit more used.

Reply:
Changed to use [https://www.npmjs.com/package/almost-equal](almost-equal)